src/examples/java/io/nats/examples/benchmark/NatsBench.java

```diff
@@ -213,10 +230,12 @@ public void run() {
                     nc.publish(subject, payload);
                     sent.incrementAndGet();
                 }
-                nc.flush(Duration.ofSeconds(5));
+                nc.flush(Duration.ofSeconds(15));
                 long end = System.nanoTime();
 
                 bench.addPubSample(new Sample(numMsgs, size, start.get(), end, nc.getStatistics()));
+
+                //System.out.println(nc.getStatistics());
```

Comment:
Extra debug statement?

Reply:
intentional in case someone (or me) wants to see advanced stats.

Reply:
ok might add a comment above stating to see more details uncomment below.

Reply:
made this a flag :-)